SU2_CFD/include/hybrid_RANS_LES_forcing.inl

```diff
@@ -91,9 +92,22 @@ inline void CHybridForcingTG0::SetTGField(
     }
   }
 
-  h[0] = A * cos(a[0]*x[0]) * sin(a[1]*x[1]) * sin(a[2]*x[2]);
-  h[1] = B * sin(a[0]*x[0]) * cos(a[1]*x[1]) * sin(a[2]*x[2]);
-  h[2] = C * sin(a[0]*x[0]) * sin(a[1]*x[1]) * cos(a[2]*x[2]);
+  // h[0] = A * cos(a[0]*x[0]) * sin(a[1]*x[1]) * sin(a[2]*x[2]);
+  // h[1] = B * sin(a[0]*x[0]) * cos(a[1]*x[1]) * sin(a[2]*x[2]);
+  // h[2] = C * sin(a[0]*x[0]) * sin(a[1]*x[1]) * cos(a[2]*x[2]);
 
+  //// CHANNEL HACK
+  //h[0] = A * cos(a[0]*x[0]) * sin(a[1]*(x[1]-1.0)) * sin(a[2]*x[2]-M_PI/2);
+  //h[1] = B * sin(a[0]*x[0]) * cos(a[1]*(x[1]-1.0)) * sin(a[2]*x[2]-M_PI/2);
+  //h[2] = C * sin(a[0]*x[0]) * sin(a[1]*(x[1]-1.0)) * cos(a[2]*x[2]-M_PI/2);
+  // // CHANNEL HACK
+  // h[0] = A * cos(a[0]*x[0]) * sin(a[1]*(x[1]-1.0)) * sin(a[2]*x[2]);
+  // h[1] = B * sin(a[0]*x[0]) * cos(a[1]*(x[1]-1.0)) * sin(a[2]*x[2]);
+  // h[2] = C * sin(a[0]*x[0]) * sin(a[1]*(x[1]-1.0)) * cos(a[2]*x[2]);
+
+  // CHANNEL HACK
+  h[0] = A * cos(a[0]*x[0]) * sin(a[1]*x[1]) * sin(a[2]*(x[2]-1.0));
+  h[1] = B * sin(a[0]*x[0]) * cos(a[1]*x[1]) * sin(a[2]*(x[2]-1.0));
+  h[2] = C * sin(a[0]*x[0]) * sin(a[1]*x[1]) * cos(a[2]*(x[2]-1.0));
```

Comment:
@trevilo Why is this shift (`x[2] - 1`) necessary?

Reply:
I don't think it is necessary.  I originally shifted z (my wall-normal coordinate) b/c Sigfried had shifted y (his wall-normal coordinate) in his first testing in CDP.  He did this for symmetry purposes, but later stopped b/c it didn't seem to matter much (or that is my understanding at least).  In any case, I don't think this shift combined with the form here actually gives the right symmetry.  It is symmetric in the wall-normal component when you would want that to be anti-symmetric, right?  This is true in CDP as well.  So, bottom line is that I think this can be safely eliminated.

Reply:
Alright.  I'll compare a short run both with and without the shift.

Reply:
The results are essentially the same:
![Figure_6](https://user-images.githubusercontent.com/13340225/64719270-8ba78680-d48d-11e9-9e00-ba920e2a807b.png)

Reply:
Looks good.